src/main/java/cloud/fogbow/common/exceptions/WrongPolicyTypeException.java

```diff
@@ -0,0 +1,21 @@
+package cloud.fogbow.common.exceptions;
+
+public class WrongPolicyTypeException extends Exception {
```

Comment:
Should it not extend Fogbow exception? If yes, it needs to be handled in FogbowExceptionToHttpErrorConditionTranslator.

Reply:
I believe this exception should only be used internally by the authorization plugins, not exposed by the API. In the case this exception is thrown by the policy, the authorization plugin should throw ConfigurationErrorException.